cves/CVE-2017-14170.yml

```diff
@@ -276,8 +315,10 @@ ipc:
     software system reads is another form of IPC.
 
     Answer should be boolean.
-  answer:
-  note:
+  answer: true
+  note: |
+    The file type involved in the vulnerability was commonly used to share video
```

Comment:
More importantly, the point you mentioned earlier about it being involved in the demuxing process is pretty crticial here.

Reply:
Added a note about that